### PR TITLE
Josiah wip

### PR DESCRIPTION
**Pull request by JosiahNonum**
added a few items (and comments) to the ingredients and recipe models

### CODE REVIEW

File: app/models/ingredient.py [hunk 1]
```diff
@@ -4,5 +4,10 @@
 
 class Ingredient(BaseModel):
     id: UUID4
+    amount: float  # maybe make a way to scale the recipe by multiplying the amount variable?
+    measurementType: str  # it should be possible to leave this blank, i.e. 1 ___ lemon vs 1 tsp cumin
```
Comment (DFNPLP) [inline app/models/ingredient.py]: Python doesn't use camel or pascal case often, the preferred/idiomatic/pythonic way of naming variables would be like "meaurement_type".

File: app/models/recipe.py [hunk 2]
```diff
@@ -1,12 +1,26 @@
...
 
 class Recipe(BaseModel):
     id: UUID4
     name: str
+    recipeYield: str
     description: str
+    ingredientList: str  # I assume this should be integrated with the ingredient model?
+
+    """
+        i'll leave the list at that for now but there are other possible items such as:
```
Comment (DFNPLP) [inline app/models/recipe.py]: Should file these as "issues" for future work, ideally todos aren't left in the code itself.

File: app/models/ingredient.py [hunk 3]
```diff
@@ -4,5 +4,10 @@
 
 class Ingredient(BaseModel):
     id: UUID4
+    amount: float  # maybe make a way to scale the recipe by multiplying the amount variable?
+    measurementType: str  # it should be possible to leave this blank, i.e. 1 ___ lemon vs 1 tsp cumin
+    # ^^^^^ maybe a shorter variable name?
```
Comment (DFNPLP) [inline app/models/ingredient.py]: That's probably fine in terms of length. You could turn the str into an enum, potentially, though.

**Review by DFNPLP — CHANGES_REQUESTED** [review]

File: app/models/recipe.py (hunk 2 above)
Comment (JosiahNonum) [reply app/models/recipe.py]: Moved those comments to the issue tab.
https://github.com/DFNPLP/CrudExcercise/issues/2

File: app/models/ingredient.py (hunk 1 above)
Comment (JosiahNonum) [reply app/models/ingredient.py]: implemented

**Pull request closed without merging**